DATASETS.md

```diff
@@ -130,13 +130,17 @@ Once you have the habitat zip archive, you should download [this SceneDatasetCon
 
 Details and download: [https://github.com/facebookresearch/Replica-Dataset](https://github.com/facebookresearch/Replica-Dataset).
 
-To work with the Replica dataset, you need a file called ```sorted_faces.bin``` for each model. Such files (1 file per model), along with a convenient setup script can be downloaded from here: [sorted_faces.zip](http://dl.fbaipublicfiles.com/habitat/sorted_faces.zip). You need:
+To work with the Replica dataset in Habitat, you will need a set of [additional configuration files](http://dl.fbaipublicfiles.com/habitat/Replica/replica_configs.zip). 
```

Comment:
Instead of putting this on S3, if they are lightweight, can we not just host them on Replica github by sending a PR?